README.md

```diff
@@ -67,25 +68,53 @@ Read more in [Why run mev-boost?](https://writings.flashbots.net/writings/why-ru
 
 # Usage
 
+## Running Beacon Node(s)
+
+- The services need access to a beacon node for event subscriptions. 
+- You can specify multiple beacon nodes by providing a comma separated list of beacon node URIs.
+  - The beacon API by default is using `localhost:3500` (the Prysm default beacon-API port).
+
+### [Option 1] Proxying [Prysm Beacon Node](https://docs.prylabs.network/docs/how-prysm-works/beacon-node)
+You can proxy the beacon-API port (eg. 3500 for Prysm) from a server like this:
+
+```bash
+ssh -L 3500:localhost:3500 your_server
+```
+
+### [Option 2] Running Lighthouse Beacon Node 
+- **NOTE:** A JSON Web Token (JWT) is required for communication between the beacon node and execution engine
+  - For details on how to generate a JWT, see [configure JWT authentication](https://docs.prylabs.network/docs/execution-node/authentication) documentation from Prysm
+```bash
+# WARNING: The command below specifies listen address for beacon node on 0.0.0.0.
+# It is NOT recommended to expose the beacon node API to the public internet. 
+lighthouse bn --network goerli --execution-endpoint http://localhost:8551 --datadir ~/.lighthouse --execution-jwt ~/.lighthouse/jwt.hex --http --http-port 5052 --http-address 0.0.0.0
+```
+
+### Running Erigon Execution Client 
+- Ethereum beacon nodes require connection to an execution client to validate block transactions
```

Comment:
remove options and reference prysm fork and builder acapella b/c of validation functionality, cut it down to a line